seed/challenges/basic-javascript.json

```diff
@@ -999,8 +1002,8 @@
         "Push the numbers 0 through 4 to <code>myArray</code> using a <code>while loop</code>."
       ],
       "tests": [
-        "assert(editor.getValue().match(/while/g), 'message: You should be using a <code>while</code> loop for this.');",
-        "assert.deepEqual(myArray, [0,1,2,3,4], 'message: <code>myArray</code> should equal <code>[0,1,2,3,4]</code>');"
+        "assert(editor.getValue().match(/while/g), 'message: You should be using a <code>while loop</code> for this.');",
+        "assert.deepEqual(myArray, [0,1,2,3,4], 'message: <code>myArray</code> should equal <code>[0,1,2,3,4]</code>.');"
```

Comment:
I would stick with `while` in the code element since `loop` isn't technically a JS keyword. Not sure how others feel about this.